plugins/woocommerce/includes/wc-core-functions.php

```diff
@@ -2591,10 +2591,11 @@ function wc_decimal_to_fraction( $decimal ) {
  *
  * @param  double $value Amount to round.
  * @param  int    $precision DP to round.
+ * @param  int    $round_mode Round mode.
  * @return float
  */
-function wc_round_discount( $value, $precision ) {
-	return NumberUtil::round( $value, $precision, WC_DISCOUNT_ROUNDING_MODE ); // phpcs:ignore PHPCompatibility.FunctionUse.NewFunctionParameters.round_modeFound
+function wc_round_discount( $value, $precision, $round_mode = WC_DISCOUNT_ROUNDING_MODE ) {
+	return NumberUtil::round( $value, $precision, $round_mode ); // phpcs:ignore PHPCompatibility.FunctionUse.NewFunctionParameters.round_modeFound
 }
```

Comment:
I feel we should put these rounding changes in another PR with testing steps that target it.

Reply:
https://github.com/woocommerce/woocommerce/pull/58918/commits/66d75cbb306a8a742193da7faa957421146cbf4f